recommender/engines/anonymous/inference/anonymous_inference_component.py

```diff
@@ -0,0 +1,75 @@
+# pylint: disable=invalid-name, too-few-public-methods
+"""Recommender engine for anonymous users"""
+
+import random
+from typing import List, Tuple, Dict, Any
+
+from recommender.services.fts import retrieve_services_for_recommendation
+from recommender.engines.panel_id_to_services_number_mapping import PANEL_ID_TO_K
+from recommender.errors import (
+    InsufficientRecommendationSpaceError,
+    InvalidRecommendationPanelIDError,
+)
+
+
+class AnonymousInferenceComponent:
+    """Engine for serving recommendations to anonymous users"""
+
+    engine_name = "anonymous"
+
+    def __init__(self, K: int) -> None:
+        self.K = K
+        if self.K not in PANEL_ID_TO_K.values():
+            raise InvalidRecommendationPanelIDError()
+
+    def __call__(self, context: Dict[str, Any]) -> List[int]:
+        """
+        This function allows for getting recommended services for the
+        recommendation endpoint based on recommendation context
+        for anonymous user.
```

Comment:
describe how the anonymous recommender chooses the recomemndations (i.e it's random)